### PR TITLE
Release v1.13.0

### PR DESCRIPTION
**Pull request by Jkovarik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [v1.3.0] - 2020-02-10
 
+### Breaking
+
 - Updated external CMA call to utilize python3 in anticipation of updates to the CMA.
 
+This is a breaking change as it requires the existence of a `python3` runtime in the environment, similar to AWS's lambda runtime for Java8
```
Comment (markdboyd) [inline CHANGELOG.md:13]: Meaning AWS lambda's runtime also requires python3?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [v1.3.0] - 2020-02-10
 
+### Breaking
```
Comment (markdboyd) [inline CHANGELOG.md]: Should we all caps and change to BREAKING CHANGES for consistency with the main repo?

Comment (Jkovarik) [inline CHANGELOG.md]:
```suggestion
### BREAKING CHANGES
```

File: CHANGELOG.md (hunk 1 above)
Comment (Jkovarik) [reply CHANGELOG.md:13]: meaning AWS's lambda runtime similarly provides python 3 as `python3`       Python installation of various python versions varies between distros/packages/etc.     

It's not uncommon to have python 3 aliased to `python`

**Review by markdboyd — APPROVED** [review]

**Pull request merged**